plugins/package-managers/maven/src/test/kotlin/utils/MavenSupportTest.kt

```diff
@@ -156,4 +160,70 @@ class MavenSupportTest : WordSpec({
             }
         }
     }
+
+    "isTychoProject()" should {
+        "return true if the Tycho extension is found" {
+            val projectDir = tempdir()
+            projectDir.addTychoExtension()
+
+            isTychoProject(projectDir) shouldBe true
+        }
+
+        "return false if the extension file does not contain the Tycho extension" {
+            val projectDir = tempdir()
+            projectDir.addTychoExtension(
+                """
+                <extensions>
+                    <extension>
+                        <groupId>org.eclipse.tycho</groupId>
+                        <artifactId>tycho-foo</artifactId>
+                        <version>4.0.0</version>
+                    </extension>
+                </extensions>
+                """.trimIndent()
+            )
+
+            isTychoProject(projectDir) shouldBe false
+        }
+
+        "return false if there is no extension file" {
+            val projectDir = tempdir()
+            val mvnDir = projectDir.resolve(".mvn")
+            mvnDir.mkdirs() shouldBe true
```

Comment:
I think it's a bit weird to assert the result of `mkdirs()` as that's not the function under test. Is this line even needed, as the point is that the file should not be existing?